src/constants.ts

```diff
@@ -1,11 +1,13 @@
-export const BUFFER_KEY_LENGTH = 64 as const;
+export const BUFFER_EXTENDED_KEY_LENGTH = 64;
 
-export const BASE_64_KEY_LENGTH = 88 as const;
+export const BUFFER_KEY_LENGTH = 32;
+
+export const BASE_64_KEY_LENGTH = 88;
 
 export const BASE_64_ZERO =
```

Comment:
If we are gonna remove the base64 parsing (see other question for whether thats intentional), some of these constants can be removed too